org-formation/050-costs/cur.yaml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  Bucket:
+    Type: String
+  Frequency:
+    Type: String
+    Default: "monthly"
+    AllowedValues:
+    - "monthly"
+    - "weekly"
+    - "hourly"
+Resources:
+  MonthlyCostReport:
+    Type: "AWS::CUR::ReportDefinition"
+    Properties:
+      ReportName: !Sub "${AWS::StackName}"
+      ReportVersioning: False
```

Comment:
[the documentation ](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cur-reportdefinition.html#cfn-cur-reportdefinition-s3prefix) says these are the possible options? `Allowed values: CREATE_NEW_REPORT | OVERWRITE_REPORT`